src/app/components/lang-selector-option/lang-selector-option.component.html

```diff
@@ -0,0 +1,5 @@
+<ng-container [ngSwitch]="lang">
```

Comment:
Please keep the line limit.

Reply:
Do not understand. What is the line limit ?

Reply:
The code line limit is 80 symbols per line.

Reply:
I see. 
but this line fits 80 limit